service/controller/v15/resource/encryptionkey/resource.go

```diff
@@ -8,81 +8,45 @@ import (
 )
 
 const (
-	// Name is the identifier of the resource.
-	Name = "kmskeyv14"
+	name = "kmskeyv14"
```

Comment:
That doesn't look right BTW.

Reply:
What is wrong?

Reply:
```
service/controller/v15/resource/encryptionkey/resource.go
```

```
kmskeyv14
```

Reply:
Should rather be `encryptionkeyv15` IMO.

Reply:
LoL, eagle eye.

Reply:
Fixed.